pkg/validation/cluster_test.go

```diff
@@ -158,3 +159,75 @@ func TestValidateUpdateWindow(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateLeaderElectionSettings(t *testing.T) {
+	tests := []struct {
+		name                   string
+		leaderElectionSettings kubermaticv1.LeaderElectionSettings
+		wantErr                bool
+	}{
+		{
+			name:                   "empty leader election settings",
+			leaderElectionSettings: kubermaticv1.LeaderElectionSettings{},
+			wantErr:                false,
+		},
+		{
+			name: "valid leader election settings",
+			leaderElectionSettings: kubermaticv1.LeaderElectionSettings{
+				LeaseDurationSeconds: pointer.Int32Ptr(int32(10)),
+				RenewDeadlineSeconds: pointer.Int32Ptr(int32(5)),
+				RetryPeriodSeconds:   pointer.Int32Ptr(int32(10)),
+			},
+			wantErr: false,
+		},
+		{
+			name: "valid leader election settings",
```

Comment:
Nit: typo.
```suggestion
			name: "invalid leader election settings",
```

Reply:
nope, this is valid for real :-)

Reply:
Is that really the case? This seems like an invalid case to me because `wantErr` is set to true and `RenewDeadlineSeconds` > `LeaseDurationSeconds`.

Reply:
you are right, I looked at the wrong line ;-)